.cirrus.yml

```diff
@@ -187,9 +189,36 @@ arch_linux_x64_stable_release_task:
 windows_server_2019_x64_stable_debug_task:
   depends_on: preflight_check
   <<: *ELK_CONTAINER_WINDOWS_SERVER_2019_X64
+  set_toolchain_script: rustup default stable
   <<: *ELK_COMMON_BUILD_AND_TEST_DEBUG
 
 windows_server_2019_x64_stable_release_task:
   depends_on: windows_server_2019_x64_stable_debug
   <<: *ELK_CONTAINER_WINDOWS_SERVER_2019_X64
+  set_toolchain_script: rustup default stable
   <<: *ELK_COMMON_BUILD_AND_TEST_RELEASE
+
+#
+# FreeBSD
+#
+
+# Pipeline 9
+
+freebsd_x64_stable_debug_task:
+  depends_on: preflight_check
+  <<: *ELK_CONTAINER_FREEBSD_X64
+  env:
+    PATH: /root/.cargo/bin:$PATH
+    HOME: /root  # must be set manually to '/root' or 'rustup' will throw an error
+  setup_script:
+    - pkg install -y git llvm
+    - pw useradd testuser1
+    - pw useradd testuser2
+    - pw groupadd testgroup1
+    - pw groupadd testgroup2
+    - mkdir -p /mnt/mqueue/
```

Comment:
Could you please add ` kldload mqueuefs` to load the mqueue kernel module. The man page states that one should do this: https://man.freebsd.org/cgi/man.cgi?query=mqueuefs